ui/perfherder/graphs/GraphsView.jsx

```diff
@@ -346,7 +346,18 @@ class GraphsView extends React.Component {
               <Col
                 className={`${testData.length ? 'graph-chooser' : 'col-12'}`}
               >
-                <Container className="graph-legend pl-0 pb-4">
+                <button
```

Comment:
Use the reactstrap Button component and then you don't need to add the type.

Reply:
Although, I suppose it doesn't matter here since it's for screen readers... :)

Reply:
I'll change it, as we are getting another update on this :)

Reply:
And sorry I always forget to add as a Reactstrap button.